libs/deal-ticket/src/components/market-info/info-market.tsx

```diff
@@ -51,7 +49,10 @@ export const MarketInfoContainer = ({ marketId }: MarketInfoContainerProps) => {
   }, []);
 
   const { data, loading, error } = useQuery(MARKET_INFO_QUERY, {
```

Comment:
chore (blocking): I realize this was already this way.. but can we add generic type args here please

Reply:
updated.